connman/src/network.c

```diff
@@ -1473,6 +1486,15 @@ bool connman_network_is_configured(struct connman_network *network,
 	return false;
 }
 
+void connman_network_set_require_all_ip_types(struct connman_network *network,
```

Comment:
I thought for this a bit because it is either setting a status variable like this or pass some appropriate `bool`/`enum connman_ipconfig_type` to define that what are required -> which would need for this case a new `enum` value. Latter option would need a change to the public plugin APIs and a lot of places. Any thoughts @inzanity @monich ?

Reply:
Obsolete. Resolving.